agents/agent-zep/agentuity-agents/agentZep/agent.py

```
@@ -0,0 +1,147 @@
+        await zep.memory.add(
+            session_id=content["session_id"],
+            messages=[Message(role_type="assistant", role="assistant", content=agent_response.choices[0].message.content or "")]
+        )
+
+        return response.text(agent_response.choices[0].message.content or "Error generating response.")
+    
+    # {"action": "addUser"} is used to add a user to the Zep database.
+    elif action == "addUser":
+        try:
+            content = data["content"]
+        except:
```

Flake8 (7.2.0)

[error] do not use bare 'except'

(E722)